app/src/main/java/com/example/touchcar/presentation/choose_model/recycler/ChooseModelAdapter.kt

```diff
@@ -0,0 +1,33 @@
+package com.example.touchcar.presentation.choose_model.recycler
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.RecyclerView
+import com.example.touchcar.databinding.ModelRecyclerItemBinding
+import com.example.touchcar.domain.entity.Model
+
+class ChooseModelAdapter() : RecyclerView.Adapter<ChooseModelViewHolder>() {
```

Comment:
Лишние скобочки у пустого конструктора
```suggestion
class ChooseModelAdapter : RecyclerView.Adapter<ChooseModelViewHolder>() {
```